src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,16 @@
+public class StringAddCalculator {
+
+	private static final int DEFAULT_RETURN_NUMBER = 0;
+
+	public static int splitAndSum(final String text){
+		SeparatedText separatedText = SeparatedText.findSeparator(text);
+		if(null == separatedText){
+			return DEFAULT_RETURN_NUMBER;
+		}
+		String[] texts = StringUtils.split(separatedText.getDelimiter(), separatedText.getTexts());
+		if(null == texts){
+			return DEFAULT_RETURN_NUMBER;
+		}
+		return StringUtils.sumWithOutNegative(texts);
```

Comment:
컨벤션에 맞춰 작성하면 동료 개발자가 코드를 파악하기 더 좋겠네요 😄

[[코딩 규칙] 자바코딩 규칙(Java Code Conventions)](https://myeonguni.tistory.com/1596)

```suggestion
		String[] texts = StringUtils.split(separatedText.getDelimiter(), separatedText.getTexts());
		
		if (null == texts) {
			return DEFAULT_RETURN_NUMBER;
		}
		
		return StringUtils.sumWithOutNegative(texts);
```

Reply:
리뷰 감사합니다. 팀 컨벤션때문에 습관화가 된 부분인데 다음부터는 알려주신 방법으로 진행해보겠습니다 ㅎㅎ